lua/lspconfig/configs/autohotkey_lsp.lua

```diff
@@ -0,0 +1,61 @@
+-- NOTE: AutoHotkey is used only on windows
+local util = require 'lspconfig.util'
+
+local function get_autohotkey_path()
+  local handle = vim.fn.exepath('autohotkey.exe')
+  local result = handle:read '*a'
+  handle:close()
+
+  result = result:gsub('%s+$', '')
+
+  return result == '' and 'autohotkey.exe' or result
```

Comment:
No idea what's going on here. exepath() returns a string, so `handle:read()` is nonsense. Did you test this code?

```suggestion
  local r = vim.fn.exepath('autohotkey.exe')
  return r == '' and 'autohotkey.exe' or result
```

Reply:
Changed the code accordingly. Sorry, didn't have enough time to look at it :)